Model/Utilities/FeatureFlags.swift

```diff
@@ -1,27 +1,19 @@
-//
-//  FeatureFlags.swift
-//  Kiwix
-//
-//  Created by Chris Li on 9/25/20.
-//  Copyright © 2020 Chris Li. All rights reserved.
-//
+//  Copyright © 2023 Kiwix. All rights reserved.
```

Comment:
Please remove everything here. If there is one header to put, this is the GPLv3 header

Reply:
opened a dedicated ticket for these here:
https://github.com/kiwix/apple/issues/575